configure.ac

```diff
@@ -34,6 +34,17 @@ AC_PROG_RANLIB
 
 AM_MISSING_PROG([GPERF], [gperf])
 
+GPERFVER=`gperf -v|head -1 |cut -d " " -f3`
```

Comment:
The version detection logic should take into account that gperf might be missing in which case the pre-generated (with 3.0.3) files from `resip/stack/gen` will be used.